pkg/installer/Dockerfile

```
@@ -44,7 +44,7 @@ ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux \
     wireless-tools libxrandr eudev-libs sudo fio iperf3 sysstat \
     lm-sensors acpi iw libdrm hwinfo dhclient e2fsprogs-extra \
     libgcc pixman libvncserver musl-utils \
```

```suggestion
    libgcc libvncserver musl-utils \
```

Do we use pixman for anything else but qemu?

---

I haven't a clue. I am happy to remove it and see if it works, although I always am afraid of breaking something. The contents are [here](https://pkgs.alpinelinux.org/contents?name=pixman&repo=main&branch=v3.21&arch=x86_64), grand total of 1 `.so` file and symlink.

---

May it be related to the interactive installer? @jsfakian?

---

We needed this for the EVE verifier. Since we don't use the verifier, we can remove it.

---

Verify script still runs, it just is automatic after installer in same image.

---

Correct, I am just saying nobody uses it :).